acceptance/localcluster/localcluster.go

```diff
@@ -137,7 +139,7 @@ func Create(numNodes int, stopper chan struct{}) *Cluster {
 		client:       newDockerClient(),
 		stopper:      stopper,
 		Nodes:        make([]*Container, numNodes),
-		UseTestCerts: true,
+		UseTestCerts: false,
```

Comment:
This isn't terribly useful now. Perhaps `UseTestCerts` should be removed. Ditto for the `TestCertGen` test.